app/wiring/AppComponents.scala

```diff
@@ -30,10 +30,10 @@ trait AppComponents extends PlayComponents {
 
   override lazy val router: Router = new Routes(
     httpErrorHandler,
-    assetController,
     applicationController,
     controllers.Default,
     monthlyContributionsController,
+    assetController,
```

Comment:
It's needed because the order that the controllers are used in the `routes` file has changed, beyond that I'm a little hazy, but @davidfurey can probably provide a more in-depth explanation.